### DISCUSSION TITLE
Alert Link broken in Kibana after upgrading to SO 2.3.4

### DISCUSSION THREAD

**Discussion by securityuser7463** [Security-Onion-Solutions/securityonion#9625] [Unsupported Versions]
After upgrading SO; when I am in Kibana (Alerts) and click on a alert the link is broken. This is the case on several of our SO deployments:

![j3FwktD38z](https://user-images.githubusercontent.com/123479363/214324474-ef007abb-628c-47e1-bd3b-e0c87a6630d1.png)

**Comment by InfosecGoon** [comment]
Unfortunately, this is an issue with field formatters in Elastic 8.5.0 that will need to be addressed with an upgrade. 

It's being tracked here: https://github.com/Security-Onion-Solutions/securityonion/issues/9594
